jdi-light-html-tests/src/test/java/io/github/epam/example/common/TitleExampleTests.java

```diff
@@ -0,0 +1,46 @@
+package io.github.epam.example.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.elements.common.Alerts.acceptAlert;
+import static com.epam.jdi.light.elements.common.Alerts.getAlertText;
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.jdiTitle;
+import static io.github.epam.html.tests.elements.BaseValidations.baseValidation;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+
+public class TitleExampleTests implements TestsInit {
+
+    String titleText = "JDI TESTING PLATFORM";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4723374)